backend/src/main/java/com/woowacourse/levellog/team/domain/TeamStatus.java

```diff
@@ -1,8 +1,30 @@
 package com.woowacourse.levellog.team.domain;
 
+import com.woowacourse.levellog.common.exception.InvalidFieldException;
+import java.util.Arrays;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
 public enum TeamStatus {
 
-    READY,
-    IN_PROGRESS,
-    CLOSED
+    READY("ready", false),
+    IN_PROGRESS("in-progress", false),
+    CLOSED("closed", true);
+
+    private final String status;
+    private final boolean isClosed;
+
+    public static boolean isClosed(final String status) {
```

Comment:
```suggestion
    public static boolean checkClosed(final String status) {
```
가 어떨까요 👀 변수명이랑 똑같아서 헷갈려요